Source/Core/Core/AchievementManager.h

```diff
@@ -15,9 +16,18 @@
 #include <rcheevos/include/rc_runtime.h>
 
 #include "Common/Event.h"
+#include "Common/HookableEvent.h"
 #include "Common/WorkQueueThread.h"
+#include "Core/Core.h"
```

Comment:
Is this still necessary in the header? Given the forward declaration below, this seems like it can be moved to the cpp file and be replaced with `#include "Common/CommonTypes.h"`

Reply:
CommonTypes isn't necessary either by the looks of things.